docs/devices/cloud-protocol/ncellmeas.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/saga/docs/devices/cloud-protocol/ncellmeas.schema.json",
+  "title": "Asset Tracker v2 neighboring cell measurement report ",
+  "description": "Describes the format which is used by the device to report neighboring cell measurements. The report is produced by the %NCELLMEAS AT command. See https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/ncellmeas.html",
+  "type": "object",
+  "properties": {
+    "mcc": {
+      "type": "integer",
+      "description": "Mobile country code. In contrast to roaming information (which uses the LinkController data), neighboring cell measurement reports contain mcc and mnc as separate integers, instead of a string.",
```

Comment:
```suggestion
      "description": "Mobile country code. In contrast to roaming information (which uses the Modem Information data), neighboring cell measurement reports contain mcc and mnc as separate integers, instead of a string.",
```

Reply:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/include/modem/modem_info.html

Link to library that combines these two values and that is used to retrieve parameters from the modem when populating "roaming" and "device" data. All data concerning neighbor cell measurements are retrieved from the link controller.